crates/sc-subspace-block-relay/src/consensus.rs

```diff
@@ -123,7 +132,7 @@ where
         &self,
         who: PeerId,
         request: BlockRequest<Block>,
-    ) -> Result<DownloadResult<BlockHash<Block>, BlockData<Block>>, RelayError> {
+    ) -> Result<Vec<u8>, RelayError> {
```

Comment:
Why not using richer type here anymore? It is always nice to limit generic `Vec<u8>` to more narrow scope.

Reply:
Changed